tools/update-client

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# This script creates a new release of the Hypothesis browser extension
+# using the latest version of the client available on npm.
+
+set -eu
+
+LATEST_CLIENT_VERSION=$(npm view hypothesis version)
+
+# Update checkout
+git checkout master
+git fetch
+git reset --hard origin/master
```

Comment:
> I think this could delete uncommitted changes.

Yes, it will. Developers have to learn somehow :P

In all seriousness, I can change this to something a bit less brutal. `git status --name --untracked-files=no` should output an empty string if there are no uncommitted changes.